src/api.js

```diff
@@ -67,11 +67,45 @@ const makeFileRequest = (fileUrl) => {
   });
 };
 
+const makeGetPaginatedRequest = (request, hasParams = false) => {
+  let returnResponse = [];
+  let toFinishLoop = false;
+
+  let separator = hasParams ? "&" : "?";
```

Comment:
Does it make sense to do this with <https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams> instead?